src/Engine.php

```diff
@@ -153,6 +153,10 @@ public function delete(string $indexName): ?array
      */
     public function search(string $query, string $indexName, array $requestOptions): array
     {
+        if ('' == $query) {
```

Comment:
```suggestion
        if ('' === $query) {
```

Reply:
You also can do something way better like:
`return $this->client->getIndex($indexName)->search(empty($query) ? null : $query, $requestOptions);`

Reply:
i was close to [suggesting that change](https://github.com/arkste/meilisearch-symfony/commit/88d20c5706554b772ed0e31ddc5d789545ca96a9), but later noticed that there's no need for `empty()` because of the `string`-typehint on `$query` (therefore its always an empty string and `'' ===` is a bit faster than `empty()`).